src/UKHO.S100PermitService.Common/Services/IPermitService.cs

```diff
@@ -4,8 +4,6 @@ namespace UKHO.S100PermitService.Common.Services
 {
     public interface IPermitService
     {
-        Task<HttpStatusCode> CreatePermitAsync(int licenceId, CancellationToken cancellationToken, string correlationId);
-
-        bool ValidateSchema(string permitXml, string xsdPath);
+        Task<(HttpStatusCode httpStatusCode, MemoryStream memoryStream)> CreatePermitAsync(int licenceId, CancellationToken cancellationToken, string correlationId);
```

Comment:
Return type here should be Task<(HttpStatusCode httpStatusCode, IStream stream)>?

Reply:
Resolved